### DISCUSSION TITLE
Question: multiple WAN failover without Speedify

### DISCUSSION THREAD

**Discussion by NaanProphet** [SmoothWAN/SmoothWAN#162] [General & Help]
Hello, I was reviewing the routing table with and without Speedify on.

From what I can tell, SmoothWAN automatically acts as a multi-WAN failover based on priority. Is this correct?

If so, the value add of using Speedify would seem to be its (1) VPN and (2) advanced bonding modes https://support.speedify.com/article/870-bonding-mode

Would appreciate any clarification. Thanks!

**Comment by TalalMash** [comment]
Linux will hop to the next lowest default metric in the routing table when the interface is down/disabled, you can change metric in LuCI web page when Speedify isn't used, however you'd still need a connectivity health checker such as the use of `mwan3` in OpenWrt to alter metric when the default WAN's internet is down (not to be confused with the interface) or if you need load balancing.

Load balancers, `mwan3`, `haproxy` auto assign new streams only, based on current load/quality, and existing streams are broken on WAN disconnections since streams exit multiple IP addresses, existing streams cannot move, they're less flexible and require good quality sources.

E.g: Having speed drop on one WAN is an issue since the LB cannot reassign existing connections to the second WAN, as a result failover isn't seamless in case of disconnection, and a lot more scenarios: packet loss, latency etc...

The server of the existing stream on the other end sees and talks to a single IP address only.

Channel bonding VPNs move the packets of the stream around multiple WANs, terminating over a single IP address (the VPN) streams don't need to be moved anymore, they can also aggregate speed for single streams hence the "bonding" term, fix packet loss with error correction (high latency and slow), deduplicate traffic across multiple WANs to fix packet loss for reduced latency compared to error correction, change QoS on existing streams, select the lowest latency WAN for specific streams such as video calls etc... 
Other than failures: cutting off one WAN when exceeding data usage limits without disruption, use the extra WAN if the speed is too low and more.

In nutshell terms: Speedify acts like a single internet connection, while a load balancer is multiple internet connection switcher.

There is more info at https://smoothwan.com -> "A note for those looking for high speed downloads"
A load balancer/failover for home users is suitable when downtimes are predictable, rare and long, while the internet quality is high on all WANs.

> From what I can tell, SmoothWAN automatically acts as a multi-WAN failover based on priority. Is this correct?

SmoothWAN doesn't do any multi-WAN features without Speedify.

**Comment by NaanProphet** [comment]
> ... when Speedify isn't used, however you'd still need a connectivity health checker such as the use of mwan3 in OpenWrt to alter metric when the default WAN's internet is down (not to be confused with the interface) or if you need load balancing

Ah! I was testing "interface" failover locally without Speedify, not internet failover. Hence why I confused myself

> [when Speedify isn't used,] existing streams are broken on WAN disconnections since streams exit multiple IP addresses, existing streams cannot move ... Channel bonding VPNs [on the other hand] move the packets of the stream around multiple WANs, terminating over a single IP address

Also makes sense now. 

> There is more info at https://smoothwan.com/ -> "A note for those looking for high speed downloads"

Thank you for all the documentation! SmoothWAN is the best!
